city_scrapers/spiders/cook_medical_examiner.py

```diff
@@ -0,0 +1,100 @@
+import datetime
+import re
+
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+from city_scrapers.constants import ADVISORY_COMMITTEE
+
+
+class CookMedicalExaminerSpider(CityScrapersSpider):
+    name = "cook_medical_examiner"
+    agency = "Cook County Medical Examiner's Advisory Committee"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.cookcountyil.gov"]
+    start_urls = ["https://www.cookcountyil.gov/service/medical-examiners-advisory-committee"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        meetings = response.css(".field-items strong *::text").getall()
+        dates = []
+        for i, item in enumerate(meetings):
+            if "Time" in item:
+                times = re.findall("(1[0-2]|0?[1-9]):([0-5]\\d)\\s*([AaPp][Mm])", item)
+                start_time = times[0]
+                end_time = times[1]
+            if "Location" in item:
+                venue = item.split(": ")[1]
+                street = meetings[i + 1] + "; " + meetings[i + 2]
+                address = {
+                    "name": venue,
+                    "address": street.replace(u'\xa0', u' '),
+                    "coordinates": {
```

Comment:
`coordinates` was part of an older version of the items returned, so it can be removed here

Reply:
Okay!